src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -1651,8 +1651,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Sub SymbolDeclaredEvent(symbol As Symbol)
             If EventQueue IsNot Nothing Then
-                Debug.Assert(Not EventQueue.IsCompleted)
-                EventQueue.Enqueue(New SymbolDeclaredCompilationEvent(Me, symbol))
+                ' In the IDE, the "My." stuff can result in attempting to declare symbols after completion.
```

Comment:
This can be reverted when Neal's change to not generate compilation events for compiler-synthesized symbols goes in.